src/Service/Admin/Posts/PostsManager.php

```diff
@@ -93,7 +93,9 @@ public function initHitsColumn()
                 add_action("manage_{$type}_custom_column", [$hitColumnHandler, 'renderHitColumn'], 10, 2);
             }
 
-            add_filter("manage_edit-sortable_columns", [$hitColumnHandler, 'modifySortableColumns']);
+            $current_page = ! empty($_GET['post_type']) ? sanitize_text_field($_GET['post_type']) : 'post';
```

Comment:
Looks good. 
Just a few things to consider:
Let's use `Request::get('post_type', 'post')` here, and also the name of the variable should be camelCase.

@hoomanhlm